doc/cephadm/services/osd.rst

```diff
@@ -1057,15 +1057,92 @@ using the `paths` keyword with the following syntax:
 Activate existing OSDs
 ======================
 
-In case the OS of a host was reinstalled, existing OSDs need to be activated
-again. For this use case, cephadm provides a wrapper for :ref:`ceph-volume-lvm-activate` that
-activates all existing OSDs on a host.
+If the operating system of a host has been reinstalled, the existing OSDs
+associated with it must be activated again. ``cephadm`` provides a wrapper for
+:ref:`ceph-volume-lvm-activate` that activates all existing OSDs on a host.
 
-.. prompt:: bash #
+The following procedure, which was developed by Eugen Block, explains how to
```

Comment:
We credit the good Mr Block below, so I suggest that we remove the text between these commas.

Reply:
I've removed this... relative clause.